tests/UKHO.S100PermitService.API.FunctionalTests/Helpers/TestConfiguration.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Microsoft.Extensions.Configuration;
+
+namespace UKHO.S100PermitService.API.FunctionalTests.Helpers
+{
+    public class TestConfiguration
+    {
+        protected IConfigurationRoot _configurationRoot;
+        public PermitServiceApiConfiguration PermitServiceConfig { get; private set; }
+
+        public class PermitServiceApiConfiguration
```

Comment:
Not sure we need a nested class here, as there is no scoping issue to resolve? Would be clearer as a namespace level declaration.